myapp/spec/models/tasks_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+describe Task, type: :model do
+  describe '#validation' do
+    describe 'title' do
+      context '0文字' do
+        let(:task) { FactoryBot.build(:task, title: '') }
+
+        it 'invalid' do
+          expect(task).to be_invalid
+        end
```

Comment:
今回のようにletがそのままテスト対象となる場合、letではなくsubjectを使いましょう。
subjectはテストの対象であることを明確にするためのものです。
また、expectの中身がsubjectだけであった場合、以下のように書くことができます。
```suggestion
        subject(:task) { FactoryBot.build(:task, title: '') }

        it 'invalid' do
          is_expected.to be_invalid
        end
```
`is_expected`を用いることでそのまま英文のように書くことができるため、
itの説明も省略して大丈夫です。（itがsubjectに相当します。）

```suggestion
        subject(:task) { FactoryBot.build(:task, title: '') }

        it { is_expected.to be_invalid }
```

Reply:
お教えいただいた通りに修正しました。
より見やすくなりました。ありがとうございます。
→評価対象の場合、letではなくsubjectにするとのこと理解しました。